test/HsLuaSpec.hs

```diff
@@ -1,10 +1,6 @@
 {-# LANGUAGE CPP #-}
 module HsLuaSpec where
 
-#if MIN_VERSION_base(4,8,0)
```

Comment:
This should not be removed but should include a test for the current lua version.  Otherwise compilation with GHC 7.8 and lua 5.1 will fail (see lines 176 ff.)